pkg/volume/util/atomic_writer.go

```diff
@@ -260,7 +259,7 @@ func validatePath(targetPath string) error {
 	if targetPath == "" {
 		return fmt.Errorf("invalid path: must not be empty: %q", targetPath)
 	}
-	if path.IsAbs(targetPath) {
+	if isAbs(targetPath) {
```

Comment:
see the question in the windows impl, I'm not sure why we want to relax this

Reply:
Removed, will treat this scenario in a separate PR: https://github.com/kubernetes/kubernetes/pull/112566